docs/developers-landing.md

```diff
@@ -4,15 +4,6 @@ title: Pulsar for Developers
 sidebar_label: "Pulsar for Developers"
 ---
 
-## Advanced topics
-
-If you want to learn how to develop Pulsar related components, check out the following guides.
-
-- [Develop simulation tools](develop-tools.md)
-- [Develop binary protocol](developing-binary-protocol.md)
-- [Develop load manager](develop-load-manager.md)
-- [Develop Pulsar plugin](develop-plugin.md)
-
 ## Popular topics
 
 If you want to read top-viewed docs for developers, check out the following concepts, examples, and tutorials.

```

Comment:
add "the" and spell out "docs" - should be "documents"

"If you want to read the top-viewed documents for...."